runner/ansible/roles/1.5.2/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.5.2'
+
+- name: "{{ test_id }}.check"
+  shell: |
```

Comment:
Cannot we use the https://docs.ansible.com/ansible/latest/collections/ansible/builtin/user_module.html module to test this?
Basically, we want to check if the `hacluster` user doesn't have `linux` as password, as this is a signal of a security hole

Reply:
> Cannot we use the https://docs.ansible.com/ansible/latest/collections/ansible/builtin/user_module.html module to test this?
It looks to me like if we could, but let's wait for an answer from an ansible expert.

Reply:
@brett060102 I have done some research and this option looks a bit nice:
(source: https://askubuntu.com/questions/611580/how-to-check-the-password-entered-is-a-valid-password-for-this-user)

```
salt=$(sudo getent shadow hacluster | cut -d$ -f3)
epassword=$(sudo getent shadow hacluster | cut -d: -f2)
match=$(python3 -c 'import crypt; print(crypt.crypt("linux", "$6$'${salt}'"))')
exit [ ${match} == ${epassword} ]
```

Could you give a try?
PD:
crypt is a native package in python3: https://docs.python.org/3.6/library/crypt.html

Reply:
OK, I new get what you are trying to do. I will take a look.

Reply:
Nice!